server/repository/src/db_diesel/user_row.rs

```diff
@@ -54,24 +48,7 @@ pub struct UserAccountRow {
     pub last_name: Option<String>,
     pub phone_number: Option<String>,
     pub job_title: Option<String>,
-    pub last_successful_sync: NaiveDateTime,
-}
-
-impl Default for UserAccountRow {
-    fn default() -> Self {
-        Self {
-            id: "".to_string(),
-            username: "".to_string(),
-            hashed_password: "".to_string(),
-            email: None,
-            language: Default::default(),
-            first_name: None,
-            last_name: None,
-            phone_number: None,
-            job_title: None,
-            last_successful_sync: Defaults::naive_date_time(),
-        }
-    }
+    pub last_successful_sync: Option<NaiveDateTime>,
```

Comment:
Made this optional so users who have never logged in before can just have a default None value for last_successful_sync